core/common/src/main/kotlin/dev/koding/catalyst/core/common/api/platform/sided/PlatformServer.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Catalyst - Minecraft plugin development toolkit
+ * Copyright (C) 2023  Koding Development
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+package dev.koding.catalyst.core.common.api.platform.sided
+
+import dev.koding.catalyst.core.common.api.platform.PlatformBinding
+import dev.koding.catalyst.core.common.api.platform.world.PlatformWorld
+
+/**
+ * The platform server class provides server specific implementations of the API.
+ * We try to keep as much in the commons module as possible, but some things are
+ * only available on the server.
+ *
+ * @author Koding
+ */
+interface PlatformServer {
+    companion object : PlatformBinding<PlatformServer>()
+
+    /**
+     * A list of all worlds on the server.
+     */
+    val worlds: List<PlatformWorld> get() = emptyList()
```

Comment:
Does the order of worlds matter? If not, why don't we use a `Set`?

Either that, or, would it not be better to use a `Map<String, PlatformWorld>`, with the key being the world's name/identifier? 

Right now, if I wanted to get the end world, right now I'd have to cycle through through all of the worlds until I find the end world, presumably through `firstOrNull`.

Reply:
Check through the class again quickly, does that resolve the issues or should I still adjust?